addons/account/models/account_move.py

```diff
@@ -241,10 +241,10 @@ def _sequence_fixed_regex(self):
         index='btree_not_null',
     )
     hide_post_button = fields.Boolean(compute='_compute_hide_post_button', readonly=True)
-    to_check = fields.Boolean(
-        string='To Check',
+    checked = fields.Boolean(
```

Comment:
When I create a new account move, this should be true - but it is not
(yes the journals `autocheck_on_post == True`)

Reply:
From benj said to me they don't want it checked on create :grimacing:

Reply:
but the logic is weird now...
If I deactivate autocheck_on_post - then I set checked = true, then post. It is to check.
and vice versa
If it will be overwritten on posting, maybe we should hide the field in draft?

Reply:
There is a button set as checked that is hidden in draft for the moment, but i agree this "feature" is weird to me also, i could put an invisible for the moment and revert it later if they want to